packages/iov-keycontrol/src/keyring.spec.ts

```diff
@@ -110,17 +111,17 @@ describe("Keyring", () => {
     const entry1 = Ed25519SimpleAddressKeyringEntry.fromEntropy(fromHex("c7f74844892fd7b707e74fc9b6c8ef917c13ddbb380cadbc"));
     const i1a = await entry1.createIdentity();
     const entry2 = new Ed25519KeyringEntry();
-    const i2a = await entry2.createIdentity();
-    const i2b = await entry2.createIdentity();
+    const i2a = await entry2.createIdentity(await Ed25519.makeKeypair(await Random.getBytes(32)));
```

Comment:
Can you move this code: `await Ed25519.makeKeypair(await Random.getBytes(32))` into a helper?

Also, maybe createIdentity could take `Promise<Ed25519Keypair> | Ed25519Keypair`
Then we could use:

`const i2a = await entry2.createIdentity(randomEd25519Keypair())`